.gitleaks.toml

```diff
@@ -4,7 +4,7 @@
   description = "Allowlist"
   # ignore secrets present in the below test folders
   paths = [
-	'''elytron\/src\/test\/resources\/org\/wildfly\/extension\/elytron''',
+	'''elytron\/subsystem\/src\/test\/resources\/org\/wildfly\/extension\/elytron''',

```

Comment:
@cam-rod  I think we can add a `test` folder of the `elytron_common` here as well so it does not notify people in the future